bulk/management/commands/data_quality.py

```diff
@@ -0,0 +1,184 @@
+    return bills
+
+
+def get_available_sessions(state):
+    return sorted(
+        s.identifier
+        for s in LegislativeSession.objects.filter(jurisdiction_id=abbr_to_jid(state))
+    )
+
+def total_bills_per_session(bills, chambers):
+    total_bills_per_session = defaultdict(list)
+    for chamber in chambers:
+        chamber_name = chamber.name.lower()
+        total_bills = bills.filter(from_organization=chamber).count()
+        # Set variables to empty strings in case any info is blank
+        latest_bill_created_id = ""
+        latest_bill_created_date = ""
+        bill_with_latest_action_id = ""
+        latest_action_date = ""
+        latest_action_description = ""
+
+        if total_bills > 0:
+            latest_bill = bills.filter(from_organization=chamber).latest("created_at")
+            latest_bill_created_id = latest_bill.identifier
+            latest_bill_created_date = latest_bill.created_at.strftime("%Y-%m-%d")
+            bill_with_latest_action = bills.filter(from_organization=chamber).latest("actions__date")
+            # In case bills don't have actions
+            if bill_with_latest_action.actions.count() > 0:
+                bill_with_latest_action_id = bill_with_latest_action.identifier
+                latest_action = bill_with_latest_action.actions.latest("date")
+                latest_action_date = latest_action.date
+                latest_action_description = latest_action.description
+
+        total_bills_per_session[chamber_name].append({
+            "chamber": chamber_name,
+            "total_bills": total_bills,
+            "latest_bill_created_id": latest_bill_created_id,
+            "latest_bill_created_date": latest_bill_created_date,
+            "bill_id_with_latest_action": bill_with_latest_action_id,
+            "latest_action_date": latest_action_date,
+            "latest_action_description": latest_action_description}
+        )
+    return total_bills_per_session
+
+
+def average_number_data(bills, chambers):
+    average_num_data = defaultdict(list)
+
+    for chamber in chambers:
+        chamber_name = chamber.name.lower()
+        total_sponsorships_per_bill = []
+        total_actions_per_bill = []
+        total_votes_per_bill = []
+
+        average_sponsors_per_bill = 0
+        average_actions_per_bill = 0
+        average_votes_per_bill = 0
+
+        for bill in bills.filter(from_organization=chamber):
+            total_sponsorships_per_bill.append(bill.sponsorships.count())
+            total_actions_per_bill.append(bill.actions.count())
+            total_votes_per_bill.append(bill.votes.count())
+
+        average_sponsors_per_bill = round(mean(total_sponsorships_per_bill))
+        average_actions_per_bill = round(mean(total_actions_per_bill))
+        average_votes_per_bill = round(mean(total_votes_per_bill))
+
+        average_num_data[chamber_name].append({
```

Comment:
looks good, maybe let's add min & max of each as well?

Reply:
Added in min & max for each. Might re-arrange how it looks in the json.